app/assets/javascripts/discourse/app/components/modal/bulk-topic-actions.gjs

```diff
@@ -16,6 +15,12 @@ import i18n from "discourse-common/helpers/i18n";
 import CategoryChooser from "select-kit/components/category-chooser";
 import TagChooser from "select-kit/components/tag-chooser";
 
+const _customActions = {};
+
+export function _addBulkDropdownAction(name, customAction) {
```

Comment:
This is very weird for a plugin API to be prefixed with `_`, that's usually for private stuff. Would just keep it `addBulkDropdownAction`

Reply:
Ya I agree, I think I saw this somewhere else in the code and just followed suit, now I don't remember. I'll update it though.